libs/navigation/api/src/lib/navigation.model.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  Field,
+  ObjectType,
+  InputType,
+  InterfaceType,
+  ArgsType,
+  OmitType,
+  registerEnumType
+} from '@nestjs/graphql'
+
+export enum NavigationLinkType {
```

Comment:
We should probably turn the schema in the DB to be an enum and get that from Prisma.	

instead of `type  String` to be `type NavigationLinkType`